public/keymaps/coseyfannitutti_discipline_default.json

```
@@ -0,0 +1,22 @@
+{
+  "keyboard": "coseyfannitutti/discipline",
+  "keymap": "default",
+  "commit": "",
```

Similarly, I put the full hash of the last commit to modify the default keymap from qmk/qmk_firmware's master branch at the time I created/modified the keymap JSON. Configurator doesn't actually use the value, but I periodically check the keymaps to make sure the ones here match those in qmk_firmware as much as possible, and having the source hash makes that easier to do.

```suggestion
  "commit": "430c37024e8de42bb735f26386f2b13f042e3569",
```